docs/core/tutorials/publishing-with-visual-studio-code.md

```diff
@@ -196,7 +196,7 @@ In the following steps, you'll look at the files created by the publish process.
 
 1. On Windows or Linux, run the app by using the executable.
 
-   1. On Windows, enter `.\HelloWorld.exe` and press <kbd>Enter</kbd>.
+   1. On Windows, enter `.\HelloWorld.exe` and press <kbd>Enter</kbd>. [On Windows with the Bash terminal, enter `.\\HelloWorld.exe']
```

Comment:
Did you intend to replace the back slash with a forward slash?
```suggestion
   1. On Windows, enter `.\HelloWorld.exe` and press <kbd>Enter</kbd>. On Windows with the Bash terminal, enter `./HelloWorld.exe`.
```

Reply:
The original single backslash does not work. The forward slash is a Unix path convention. The backslash is used in Window's paths as the use of the forward slash in Windows is mostly for command line parameters. The suggestion to use a second backslash seems like a reasonable fix and is consistent with Windows use.  Since the direction is specifically for Windows it's probably best to use Windows conventions. Most Unix users are aware of this and will not be confused.

Reply:
I see the confusion. The browser is interpreting the backslash as an escape and only displaying a single backslash. In order to have two backslash characters I needed to put three! Most likely this is what is happening on the tutorial.

Reply:
After looking at this again, Bash is part of Unix and probably better to stay with those conventions even when running it in Windows.